ziti/cmd/verify/ops_verify_network.go

```diff
@@ -48,7 +49,7 @@ type stringMapList []interface{}
 
 type StringMap map[string]interface{}
 
```

Comment:
Why add this, just to ignore it?

Reply:
meh - it was just a consistency thing... all the other commands pass these params so it was consistent. i had it with no params, but then verify-traffic needed in/out streams...  so -- yeah that's 'why'